proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+  <param name="headSupportVerticalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  
+  <param name="massageEnabled" type="Boolean" mandatory="false"></param>
+  <param name="massageMode" type="MassageModeData" minsize="1" maxsize="2" array="true" mandatory="false"></param>
+  <param name="massageCushionFirmness" type="MassageCushionFirmness" minsize="1" maxsize="5" array="true" mandatory="false"></param>
+  
+  <param name="memory" type="SeatMemoryAction" mandatory="false"></param>
+</struct>
+
+<struct name="SeatControlCapabilities">
+  <param name="moduleName" type="String" maxlength="100"  mandatory="true">
+    <description>
+    The short friendly name of the light control module.
+    It should not be used to identify a module by mobile application.
+    </description>
+  </param>
+  <param name="heatingEnabledAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="coolingEnabledAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="heatingLevelAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="coolingLevelAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="horizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="verticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="frontVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backTiltAngleAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="headSupportHorizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="headSupportVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageEnabledAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageModeAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageCushionFirmnessAvailable" type="Boolean" array="false">
+  </param>
+  <param name="memoryAvailable" type="Boolean" array="false">
+  </param> 
+</struct> 
+
+<struct name="RemoteControlCapabilities">
+  :
+  :
+  <param name="seatControlCapabilities" type="SeatControlCapabilities" mandatory="false" minsize="1" maxsize="100" array="true">
+    <description>
+      If included, the platform supports seat controls.
+    </description >
+  </param>
+</struct>
+
+<struct name="ModuleData">
+  :
+  :
+  <param name="seatControlData" type="SeatControlData" mandatory="false">
+  </param>
+</struct>
+```
+#### Additions to HMI_API
+The changes are the same as mobile api changes, they are not duplicated here. 
+
+## Potential downsides
+
+Due to the lack of an agreed control resource ID/Zone scheme, we define `SupportedSeat` enum as a work around to allow control both driver seat and front passenger seat.
```

Comment:
Please update "...to allow control both driver seat..." to "...to allow control of both driver seat..."

Reply:
changed,  thank you